### DISCUSSION TITLE
Can user convert Issue into Discussion without rewrite it ?

### DISCUSSION THREAD

**Discussion by Nick222** [zim-desktop-wiki/zim-desktop-wiki#1664] [Q&A] [answered]
Can user convert Issue into Discussion without rewrite it ?

**Comment by jaap-karssenberg** [comment, marked as answer]
Not sure if every user can do it, but moderators for sure can do this. So
feel free to comment on items that should be a discussion instead and I can
convert them.

**Reply by introt** [reply]
Non-mods can't. In the other direction, anyone can create an issue from a discussion. The `transfer` action is for moving between repositories and isn't related (was wondering how'd it handle transferring comment threads).
